src/ImageGallery.react.jsx

```diff
@@ -351,6 +352,7 @@ export default class ImageGallery extends React.Component {
                       onClick={this._wrapClick(swipeNext)}/>
                   ],
                 <Swipeable
+                  className='image-gallery-swipe'
```

Comment:
In some special case,  I need to select the swipeable div to implement the UI. I think it is OK to add a class name for it.